sftp.go

```diff
@@ -185,9 +185,9 @@ func unimplementedPacketErr(u uint8) error {
 	return fmt.Errorf("sftp: unimplemented packet type: got %v", fxp(u))
 }
 
-type unexpectedIdErr struct{ want, got uint32 }
+type unexpectedIDErr struct{ want, got uint32 }
```

Comment:
There are several other errors like this in the same file.  I can't think of a better name at the moment, but perhaps it would be best to address them all as a unit in a second pull request.